plat/kvm/efi.c

```diff
@@ -37,6 +41,21 @@ static __u8 uk_efi_mat_present;
 #define EFI_STUB_INITRD_FNAME	CONFIG_KVM_BOOT_PROTO_EFI_STUB_INITRD_FNAME
 #define EFI_STUB_DTB_FNAME	CONFIG_KVM_BOOT_PROTO_EFI_STUB_DTB_FNAME
 
+#define UK_EFI_MAX_FMT_STR_LEN					256
+
+#if CONFIG_LIBUKDEBUG_PRINTD
+#define uk_efi_pr_debug					uk_efi_printf
+/* UEFI for proper \n, we must also use CRLF */
+#define UK_EFI_CRASH(...)					\
+	do {							\
+		uk_efi_printf(__VA_ARGS__);			\
+		uk_efi_do_crash();				\
+	} while (0)
+#else
```

Comment:
By the way we have a sort of styling revamp process going on. The way we do `#if`'s is something like:
```C
#if CONFIG
...
#endif /* CONFIG */
```
and in your case
```C
#if CONFIG
...
#else /* !CONFIG */
...
#endif /* !CONFIG */
```